src/main/java/com/mabl/integration/jenkins/MablRestApiClientImpl.java

```diff
@@ -311,7 +312,7 @@ private RequestConfig getDefaultRequestConfig() {
                 .setConnectTimeout(REQUEST_TIMEOUT_MILLISECONDS)
                 .setConnectionRequestTimeout(REQUEST_TIMEOUT_MILLISECONDS)
                 .setSocketTimeout(REQUEST_TIMEOUT_MILLISECONDS)
-                .setProxy(proxy)
+                .setProxy(proxy != null ? new HttpHost(proxy.name, proxy.port) : null)
                 .setProxyPreferredAuthSchemes(Collections.singletonList(AuthSchemes.BASIC))
```

Comment:
Curious, are we assuming all proxies will use base auth? Perhaps that's the only format supported by Jenkins plugins?

Reply:
Yes - that's the only way I found it in existing plugins. Some don't even support proxy authentication.